src/backend/distributed/commands/type.c

```diff
@@ -82,7 +82,7 @@
 
 
 /* guc to turn of the automatic type distribution */
-bool EnableCreateTypePropagation = true;
+bool EnableCreateTypePropagation = false;
```

Comment:
why change this?

Reply:
I guess during testing he changed and  forgot? @halilozanakgul let's not forget to undo this